docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -10,7 +10,7 @@ Reference of interfaces, presets, and utilities related to account contracts.
 
 [.contract]
 [[ISRC6]]
-=== `++ISRC6++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.8.0-beta.0/src/account/interface.cairo#L12[{github-icon},role=heading-link]
+=== `++ISRC6++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.8.0-beta.0/src/account/interface.cairo[{github-icon},role=heading-link]
```

Comment:
don't we want to update the line number? by the way, these links are broken across the API page, in `Account Component` and `Preset`.

I also realized `IDeclarer`, `IDeployer`, as well as camel only traits are not documented, only the impls

Reply:
> I also realized IDeclarer, IDeployer, as well as camel only traits are not documented, only the impls.

Do you think it is worth adding them? That would make the API Reference grow bigger without adding much real value IMO (in this particular case).

Reply:
> I think ew shouldn't, because that's very hard to maintain, when updating a file we would need to look in the doc if a line was affected, and I don't think is worth the benefit. Maybe I'm wrong.

True, I wonder if there's a way of automating it, but without the line to me it makes no sense to link in each section. I think the best experience is to have direct links, probably they should not be updated manually in each PR, we could:

- check for link health just once before each release, still manual but way less friction and contained
- build some tool/hook to detect that a linked line of code got modified in a commit and alert us so we update it
- both

Reply:
> Do you think it is worth adding them? That would make the API Reference grow bigger without adding much real value IMO (in this particular case).

I'm not sure. I don't think it'll grow much since I expect most of it to be moved from the component and the component would look similar to the preset. I don't have strong feelings, I just think it's weird to not mention it since it's referenced (e.g. in the impls). I guess it's good for users to know they're available.

Reply:
> Are there other broken links?
All the source code links, they're not _broken links_, they're just pointing to the wrong lines.